airflow/timetables/events.py

```diff
@@ -28,25 +28,38 @@ class EventsTimetable(Timetable):
     Timetable that schedules DAG runs at specific listed datetimes. Suitable for
     predictable but truly irregular scheduling such as sporting events.
 
-    :param event_dates: List of datetimes for the DAG to run at
+    :param event_dates: List of datetimes for the DAG to run at. Duplicates will be ignored.
```

Comment:
Should also mention that this iterable is consumed eagerly, so it must be finite and of a reasonable size.